dao/src/main/java/org/thingsboard/server/dao/audit/AuditLogServiceImpl.java

```diff
@@ -382,7 +382,11 @@ private ListenableFuture<List<Void>> logAction(TenantId tenantId,
         AuditLog auditLogEntry = createAuditLogEntry(tenantId, entityId, entityName, customerId, userId, userName,
                 actionType, actionData, actionStatus, actionFailureDetails);
         log.trace("Executing logAction [{}]", auditLogEntry);
-        auditLogValidator.validate(auditLogEntry, AuditLog::getTenantId);
+        try {
+            auditLogValidator.validate(auditLogEntry, AuditLog::getTenantId);
+        } catch (Exception e) {
+            return Futures.immediateFailedFuture(e);
```

Comment:
I strongly believe we should store the audit log, but replace the entityName or userName with "malformed".